src/models/StyledNativeComponent.js

```diff
@@ -95,8 +96,21 @@ export default (constructWithOptions: Function) => {
       }
     }
 
+    setNativeProps(nativeProps: Object) {
```

Comment:
We need ```setNativeProps``` to forward nativeProps to the underlying native component. The reason of this PR is that it's missing from v2.